reports/Michigan.md

```diff
@@ -129,6 +129,20 @@ id: mi-detroit-9
 * https://twitter.com/DJEazyTwist/status/1277430912653950976
 
 
+### Riot police officer chokes protestor. Tear gas was also used. | July 11th
+
+A Police officer in full riot gear is seen arresting Detroit Will Breathe organizer, Nakia Renne Wallace by Chokehold. It was reported another individual was arrested by force with a knee to the neck. Nonviolent protestors were dispersed using shields, batons, and tear gas.
```

Comment:
Probably lowercase "Chokehold" here.

Might be good to mention some of the surrounding circumstances (bottles & rocks thrown). Certainly doesn't excuse the police behavior, but it can help get ahead of any "well they deserved it" comments by being transparent